codec-http/src/main/java/io/netty/handler/codec/http/HttpServerUpgradeHandler.java

```diff
@@ -61,9 +61,10 @@
 
         /**
          * Adds any headers to the 101 Switching protocols response that are appropriate for this protocol.
+         * Returns {@code true} if an upgrade was done, {@code false} otherwise.
```

Comment:
`Performs the preparatory steps required for performing a protocol update.` -> `Prepares the {@code upgradeHeaders} for a protocol update based upon the contents of {@code upgradeRequest}`.

"Perform" is used multiple times ... consider being more explicit upfront.